lib/kubernetes-deploy/runner_task.rb

```diff
@@ -17,7 +17,7 @@ class TaskTemplateMissingError < TaskConfigurationError; end
 
     def initialize(namespace:, context:, logger: nil, max_watch_seconds: nil)
       @logger = logger || KubernetesDeploy::FormattedLogger.build(namespace, context)
-      @task_config = KubernetesDeploy::TaskConfig.new(context, namespace, @logger)
+      @task_config = KubernetesDeploy::RunnerTaskConfig.new(context, namespace, @logger)
```

Comment:
The idea behind the TaskConfig was an object we could pass to other classes instead of a bunch of individual params. The fact that the only class that uses the task_template & args is the validator is making me wonder if we should just pass those two values to the Validator instead of create a new RunnerTaskConfig. Not being able to fully populate the RunnerTaskConfig until `Run!` seems odd, but might not really be an issue.

Reply:
I removed `RunnerTaskConfig` and added those two values to the validator. Thanks for the context!